app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -144,6 +149,8 @@
     private TextToSpeech textToSpeech;
     private String[] array;
     private String timenow;
+    private int remcount1 ;
```

Comment:
The variable name could be a bit more descriptive?
Because since this is an open Source project, others will also be contributing to this project hence either making the variable names descriptive or adding comments specifying what the variable is meant for makes is easy for others to understand the codebase.